doc/specs/top_10.md

```diff
@@ -0,0 +1,92 @@
+# Top 10
```

Comment:
Rather then calling it top 10 we will call it "Charts" as this is generic term for any of the content appearing on these page(s)